unitTests/UnitTest.cs

```diff
@@ -10,11 +14,17 @@ public class UnitTest
     {
         private Calculator calculator { get; }
```

Comment:
![image](https://user-images.githubusercontent.com/35724303/164515066-9bebde61-76fc-406c-bf1e-1484c6de2537.png)
These should not be in the commit.